### DISCUSSION TITLE
Roadmap Page?

### DISCUSSION THREAD

**Discussion by libbyslayer** [ente-io/ente#3684] [General]
can we have dedicated ROADMAP page in Ente website? this would help existing users not raise feature requests and new users to migrate if the feature they want is already in pipeline.

**Comment by hknoener** [comment]
Does this silence imply a negative?

**Comment by whoami730** [comment]
Even if a roadmap page is kinda hard, maybe tags to indicate which of the features/improvements have been planned, which are in progress and which will not be considered.

**Comment by cfi700** [comment]
I also think a roadmap is useful, as there are a lot of features in the discussion section that have been strongly requested by many users. This would allow users to decide whether they want to wait for the feature or perhaps need a different app.

**Comment by vishnukvmd** [comment]
We recently shared our thoughts around this in our community AMA: https://youtu.be/eRy3E-_5KS4?t=470

TL;DW: You're welcome to ask the status of features over Discord or email, we'd be happy to share. We are not fans of dealing with the overhead that comes with project / roadmap management.

**Reply by whoami730** [reply]
Sounds reasonable. Though I believe https://github.com/ente-io/ente/discussions/3684#discussioncomment-12745659 can be looked at?

**Reply by vishnukvmd** [reply]
We'd rather not label a suggestion as forever out of scope.

**Reply by whoami730** [reply]
Planned / In-progress?

**Reply by whoami730** [reply]
Like how mozilla does it (https://connect.mozilla.org/t5/ideas/idb-p/ideas/tab/most-kudoed)
